scripts/data_files/vs2013-app-template.vcxproj

```diff
@@ -99,7 +99,7 @@ INCLUDE_DIRECTORIES
     <Link>
       <SubSystem>Console</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
-      <AdditionalDependencies>kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>bcrypt.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Non-blocker, but I didn't mean to remove these from the template where they were, just to not add them to the template where they were not already included. Again, I can't test this, so have to presume they are still working.

Reply:
To be fair they were added to the old template and we have moved to vs2013 which singifincaly changed those files. Looking into it I could not see any specific reason as to why they should be needed. I have manually tested the simpler version with vs2022 and it works, and the CI is clearing. 

If we need something specific dependencies linking we can add it later on.